daml-lf/transaction/src/main/protobuf/com/daml/lf/transaction.proto

```diff
@@ -45,6 +45,7 @@ message Node {
         NodeFetch fetch = 4;
         NodeExercise exercise = 5;
         NodeLookupByKey lookup_by_key = 6;
+        NodeRollback rollback = 7;
```

Comment:
please add the version in a comment like we do for other stuff in this file.

Reply:
done